jobs/JEVS_GLOBAL_ENS_CHEM_GRID2OBS_PLOTS

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+########################################
+# Preliminary data setup step
+########################################
+set -xa
```

Comment:
@Ho-ChunHuang-NOAA @malloryprow Can `set -xa` be changed to `set -x` to be consistent across EVS?

Reply:
done